geoportal/geoportailv3_geoportal/views/authentication.py

```diff
@@ -88,9 +88,9 @@ class O(object):
                 if 'roleOGC' in obj:
                     user.ogc_role = int(obj['roleOGC'][0])
         try:
-            user.role = DBSession.query(Role).filter_by(id=user.mymaps_role).one()
```

Comment:
@rmichaelis, what is the meaning of `roleTheme`?

Reply:
@gberaudo we have several roles associated with a single user. 
"mymaps_role" associates a user  with a role dedicated to "mymaps" 
"ogc_role"  associates a user  with a role dedicated to wms stuffs.
"roletheme" associates a user with a theme.